src/identity/PhoneNumberLookupQuotaScreen.tsx

```diff
@@ -15,14 +14,15 @@ import { LOOKUP_GAS_FEE_ESTIMATE } from 'src/identity/privateHashing'
 import { isUserBalanceSufficient } from 'src/identity/utils'
 import { Screens } from 'src/navigator/Screens'
 import { StackParamList } from 'src/navigator/types'
-import { cUsdBalanceSelector } from 'src/stableToken/selectors'
 import fontStyles from 'src/styles/fonts'
+import { useTokenInfoByCurrency } from 'src/tokens/hooks'
+import { Currency } from 'src/utils/currencies'
 
 type Props = NativeStackScreenProps<StackParamList, Screens.PhoneNumberLookupQuota>
 
 function PhoneNumberLookupQuotaScreen(props: Props) {
   const [isSending, setIsSending] = useState(false)
-  const userBalance = useSelector(cUsdBalanceSelector) ?? null
+  const userBalance = useTokenInfoByCurrency(Currency.Dollar)?.balance.toNumber() ?? null
```

Comment:
So we don't loose precision:
```suggestion
  const userBalance = useTokenInfoByCurrency(Currency.Dollar)?.balance.toString() ?? null
```